### DISCUSSION TITLE
SAE deinstallieren

### DISCUSSION THREAD

**Discussion by CHB68** [camueller/SmartApplianceEnabler#519] [Installation und Betrieb]
Ich habe sae manuell auf meinem Raspberry installiert auf dem auch andere produktive Dienste laufen. Leider hängt sich der raspberry seit der Installation von SAE ständig auf sodass man ihn neu starrten muss. Eine Fehleranalyse habe ich nicht hin bekommen. Das System bleibt aber stabil, wenn SAE nicht läuft. Ich ziehe SAE auf ein anderes System um. 
Leider bekommen ich SAE nicht sauber deinstalliert. Nach einem Systemupdate und reboot läuft sae wieder automatisch. Ein
sudo systemctl disable smartapplianceenabler
scheint nicht zu reichen. kann jemand helfen?

**Comment by camueller** [comment]
Ob der SAE allein oder mit anderen Diensten läuft, hat auf die Stabilität keinen EInfluss, solange genug Resourcen (CPU, Speicher) vorhanden sind.

Wenn der SAE nach `sudo systemctl disable smartapplianceenabler` und eine Reboot trotzdem läuft, scheint Deine gesamte Systeminstallation kaputt zu sein. Da wird Dir wohl niemand helfen können.

**Reply by CHB68** [reply]
Danke für die Antwort. Ich möchte auch nicht den Eindruck hinterlassen, dass es an deiner wirklich tollen Software liegt. Es scheint sich um ein Spezialprogramm meines Setups z7 handeln und der einfachste Weg für mich ist, SAE auf einem anderen System zu hosten.
Nach einem reboot alleine ist SAE bisher auch deaktiviert geblieben. Das es jetzt plötzlich wieder aktiv war, lag vermutlich an dem Systemupdaten des Raspi vor einigen Tagen (sudo apt-get update und upgrade).
